test/cctest/test_url.cc

```diff
@@ -109,3 +109,36 @@ TEST_F(URLTest, ToFilePath) {
 
 #undef T
 }
+
+TEST_F(URLTest, FromFilePath) {
+  URL file_url;
+#ifdef _WIN32
+  file_url = URL::FromFilePath("C:\\Program Files\\");
+  EXPECT_EQ(file_url.protocol(), "file:");
+  EXPECT_EQ(file_url.path(), "/C:/Program%20Files/");
+
+  file_url = URL::FromFilePath("C:\\a\\b\\c");
+  EXPECT_EQ(file_url.protocol(), "file:");
+  EXPECT_EQ(file_url.path(), "/a/b/c");
+
+  file_url = URL::FromFilePath("b:\\a\\%%.js");
+  EXPECT_EQ(file_url.protocol(), "file:");
+  EXPECT_EQ(file_url.path(), "/a/%25%25.js");
+
+  file_url = URL::FromFilePath("\\host\\a\\b\\c");
+  EXPECT_EQ(file_url.protocol(), "file:");
+  EXPECT_EQ(file_url.path(), "host/a/b/c");
```

Comment:
It should be `"\\\\host\\a\\b\\c"`. The output file URL should be `file://host/a/b/c` (notice how there are only two slashes after `file:`). I'm fine with just removing this test for now and file an issue to make it work with both `URL::FromFilePath()` and the JavaScript version of the function later.

Reply:
I replaced the start with \\\\ and this test by itself looks good to me, since EXPECT_EQ for path checks that path does not contain leading slashes. We may check it better when href method will be implemented.

Reply:
removed for now.